### DISCUSSION TITLE
State of Persistent Restarts

### DISCUSSION THREAD

**Discussion by ghnp5** [memcached/memcached#1145] [General]
Hello!
Thank you so much for Memcached. :)

I'd like to know, please - what's the current state of Persistent Restarts?

It's been a while since it was released, and I'd like to finally start using it, but when I look here:  
https://github.com/memcached/memcached/wiki/WarmRestart

It still says:
> Consider this feature experimental for the next few releases

And, does this work well with Docker too? (using the `memcached` image in DockerHub)

However, though... if we use a tmpfs location, memcached will still be cleared when rebooting the host, am I right?

Many thanks!

**Comment by dormando** [comment]
So far as I know it works fine; nobody has filed any complaints. The only major issue is it doesn't work with extstore.

I have no idea if it works with the docker image: your docker image would need tmpfs mounted from an external volume at least.

Yes, rebooting the host would clear it. You can copy the file to disk and back into tmpfs and restore it after reboot, but it can take a long time if you have large memory and slow disk.

**Reply by ghnp5** [reply]
Thank you very much. That's a good enough answer!
